sdk/clientcore/core/src/test/java/io/clientcore/core/implementation/http/client/JdkHttpClientIT.java

```diff
@@ -169,7 +169,7 @@ public void testConcurrentRequests() throws InterruptedException {
         int numRequests = 100; // 100 = 1GB of data read
         HttpClient client = new JdkHttpClientBuilder().build();
 
-        ForkJoinPool pool = new ForkJoinPool();
+        ForkJoinPool pool = new ForkJoinPool((int) Math.ceil(Runtime.getRuntime().availableProcessors() / 2.0));
```

Comment:
Can we put this in a field on `TestBase` or something? It's cheap enough to compute but having it central seems better.